# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../spiralsynthmodular/param.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# Copyright (C) 1998 - 2003 ROCK Linux Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- ./SpiralSound/Plugins/MeterPlugin/MeterPluginGUI.C.org	Mon Jan 20 14:18:44 2003
+++ ./SpiralSound/Plugins/MeterPlugin/MeterPluginGUI.C	Mon Jan 20 14:19:05 2003
@@ -56,7 +56,7 @@
    MaxBox->box (FL_ENGRAVED_BOX);
    MaxBox->color (16);
    add (MaxBox);
-   Meter = new Fl_VU_Meter (2, 62, 226, 20);
+   Meter = new Fl_VU_Meter (2, 62, 226, 20, NULL );
    Meter->color (FL_BLACK);
    cb_Reset_i (Reset, NULL);
    add (Reset);
